src/Elasticsearch.Net/ConnectionPool/SniffingSortedStickyConnectionPool.cs

```diff
@@ -5,74 +5,31 @@
 
 namespace Elasticsearch.Net
 {
-	public class SniffingSortedStickyConnectionPool : IConnectionPool
+	public class SniffingSortedStickyConnectionPool : SniffingConnectionPool
```

Comment:
Awesome :+1: ty for taking on more then initially bargained for!

Reviewing first thing in the morning